src/components/ReportActionsSkeletonView/index.js

```diff
@@ -32,10 +32,15 @@ const ReportActionsSkeletonView = (props) => {
                 skeletonViewLines.push(<SkeletonViewLines animate={props.animate} numberOfRows={1} key={`skeletonViewLines${index}`} />);
         }
     }
-    return <>{skeletonViewLines}</>;
+    return (
+        <>
+            {skeletonViewLines}
+        </>
+    );
 };
 
 ReportActionsSkeletonView.displayName = 'ReportActionsSkeletonView';
 ReportActionsSkeletonView.propTypes = propTypes;
 ReportActionsSkeletonView.defaultProps = defaultProps;
+
```

Comment:
Can you please undo the changes in this file? They don't make any material difference and it's just extra noise.